ui/appui-react/src/appui-react/pickers/ViewSelector.tsx

```diff
@@ -341,6 +357,11 @@ export class ViewSelector extends React.Component<ViewSelectorProps, ViewSelecto
         items={this.state.items}
         iconSpec={"icon-saved-view"}
         onExpanded={this._onExpanded}
+        searchBox={this.state.searchBox}
+        onSearchValueChange={(search: string) => {
+          this._searchInput = search;
+          void this.loadViews();
```

Comment:
If we are not waiting for the results, we probably want to add a `debounce` here, because the results may not come in the same order (so the "last" render might be the next to last character typed) and the speed at which the user types will impact these results (in case the views list need to go to the backend for some reasons...)

We should already have use of debounce/throttle somewhere in the code.